interview-questions.md

```diff
@@ -9,43 +9,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is a hash in Ruby?
 
 Your answer:
+A hash in Ruby is like an array in JavaScript because it stores data in key-value pairs, but hashes don't use numbers as an index.
 
 Researched answer:
+Hashes can also be look at as dictionaries because they perform like them. The term iterate and its definition are an example of a hash:
```

Comment:
Cool. More content can be sharing ruby hash to support this dictionary connection